lib/synapse/service_watcher/zookeeper.rb

```diff
@@ -35,6 +42,12 @@ def initialize(opts={}, synapse)
           @decode_method = self.method(:serverset_decode)
         end
       end
+
+      @retry_policy = @discovery['retry_policy'] || {}
```

Comment:
just want to make sure that you want to exponential backoff on by default.

Reply:
no retry by default. if `retry_policy` is not in the config, @retry_policy will be empty, `with_retry` by default set `max_attempts` to ` 1, so only original call will be attempted.